tooling/noir_js_backend_barretenberg/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@noir-lang/backend_barretenberg",
+  "collaborators": [
+    "The Noir Team <team@noir-lang.org>"
+  ],
+  "version": "0.7.10",
+  "packageManager": "yarn@3.5.1",
+  "license": "(MIT OR Apache-2.0)",
+  "type": "module",
+  "source": "src/index.ts",
+  "main": "lib/cjs/index.cjs",
+  "module": "lib/esm/index.js",
+  "exports": {
+    "require": "./lib/cjs/index.cjs",
+    "default": "./lib/esm/index.js",
+    "types": "./lib/esm/index.d.ts"
+  },
+  "types": "lib/esm/index.d.ts",
+  "scripts": {
+    "dev": "tsc --watch",
+    "build": "yarn clean && tsc && tsc -p ./tsconfig.cjs.json && mv ./lib/cjs/index.js ./lib/cjs/index.cjs && mv ./lib/cjs/serialize.js ./lib/cjs/serialize.cjs && mv ./lib/cjs/base64_decode.js ./lib/cjs/base64_decode.cjs",
```

Comment:
This is not great, might be better to patch the artefacts with a package.json file in the future